test/sdc-api.spec.js

```diff
@@ -40,8 +52,8 @@ describe('API calls', () => {
         downloadableResources: 20,
       },
       previewVideo: {
-        previewVideoImgUrl: 'http://localhost:3004/assets/previewVideoImg6.jpg',
-        previewVideoUrl: 'http://localhost:3004/assets/previewVideo6.mp4',
+        previewVideoImgUrl: 'videoImg/1.jpg#116378',
```

Comment:
what is the hash at the end for?  Is that a normal convention?

Reply:
It is mostly to make the DB think that there are unique links so that it doesn't do any automatic caching. I think i just added it here so everything would look the same.